module/documents/activity/summon.mjs

```diff
@@ -40,7 +40,9 @@ export default class SummonActivity extends ActivityMixin(SummonActivityData) {
   /** @inheritDoc */
   static localize() {
     super.localize();
-    this._localizeSchema(this.schema.fields.profiles.element, ["DND5E.SUMMON.FIELDS.profiles"]);
+    if ( game.release.generation < 13 ) {
```

Comment:
Need to backport these to 4.3.7

Reply:
https://github.com/foundryvtt/dnd5e/pull/5420